packages/api/cms-api/src/user-permissions/dto/user-permission.input.ts

```diff
@@ -0,0 +1,48 @@
+import { Field, ID, InputType, PartialType } from "@nestjs/graphql";
+import { IsDate, IsOptional, IsString, IsUUID } from "class-validator";
+
+@InputType()
+export class UserPermissionInput {
```

Comment:
This base input isn't used anywhere, so we don't need to export it. You may also call it BaseUserPermissionInput

Reply:
fixed